api/presenters/PostPresenter.js

```diff
@@ -62,10 +64,17 @@ var postAttributes = (post, opts) => {
       public: post.get('visibility') === Post.Visibility.PUBLIC_READABLE
     })
   if (opts.withComments) {
-    console.log('COMMENTS', rel.comments)
+    extendedPost.comments = rel.comments.map(c => _.merge(
+        c.pick('id', 'text', 'created_at', 'user'),
```

Comment:
I'd prefer to keep numComments for now, as we will likely soon not be returning the full list of comments with the post, in which case we'll still need the count.